dm/dumpling/dumpling_test.go

```diff
@@ -124,3 +124,30 @@ func (d *testDumplingSuite) TestDefaultConfig(c *C) {
 	c.Assert(dumpling.dumpConfig.StatementSize, Not(Equals), export.UnspecifiedSize)
 	c.Assert(dumpling.dumpConfig.Rows, Not(Equals), export.UnspecifiedSize)
 }
+
+func (d *testDumplingSuite) TestCallStatus(c *C) {
+	dumpConf := export.DefaultConfig()
+	taskCfg := &config.SubTaskConfig{}
+	m := &Dumpling{cfg: taskCfg}
+	ctx := context.Background()
+
+	s := m.Status(nil).(*pb.DumpStatus)
+	c.Assert(s.CompletedTables, Equals, float64(0))
+	c.Assert(s.FinishedBytes, Equals, float64(0))
+	c.Assert(s.FinishedRows, Equals, float64(0))
+	c.Assert(s.EstimateTotalRows, Equals, float64(0))
+
+	dumpling, err := export.NewDumper(ctx, dumpConf)
+	c.Assert(err, NotNil)
```

Comment:
```suggestion
	c.Assert(err, IsNil)
```

Reply:
this error is `sql: SELECT version();: Error 1045: Access denied for user 'root'@'localhost' (using password: NO)`

i don't think our ut should access real mysql, could please try fix this error